clearblade.md

```diff
@@ -1288,9 +1288,15 @@ Class: ClearBlade.Database();
 This class allows you to specify the query in raw SQL on platform and edge instead of using the existing ClearBlade query model. The function can be used for operations, such as `JOIN` and `SOME`, that are not supported by the ClearBlade Query Model.
 
 ~~~javascript
+//Issues commands for external collections
```

Comment:
Instead of adding a comment here. Add an explanation above the code snippet. 

Use the Database object without the options object for performing query and exec operations on collections.